blocks/extensions/__init__.py

```diff
@@ -506,23 +506,35 @@ def before_batch(self, batch):
 
 
 class Timing(SimpleExtension):
-    """Add timing information to the log.
+    r"""Add timing information to the log.
 
     This adds data about the time spent in the algorithm's
     :meth:`~.Algorithm.process_batch` method as well as the time spent
     reading data per batch or epoch. It also reports the time spent
     initializing the algorithm.
 
+    Parameters
+    ----------
+    prefix : str
+        Prefix to be added to the log record, looks the best if ends with
+        ``_``. Defaults to the empty string.
```

Comment:
Hmm, I think in other places we add the "_" automatically, e.g. DataStreamMonitoring